core/src/main/kotlin/com/tschuchort/compiletesting/KotlinCompilation.kt

```diff
@@ -387,10 +390,15 @@ class KotlinCompilation : AbstractKotlinCompilation<K2JVMCompilerArguments>() {
 				putIfAbsent(OPTION_KAPT_KOTLIN_GENERATED, kaptKotlinGeneratedDir.absolutePath)
 			}
 
-			it.mode = AptMode.STUBS_AND_APT
+            it.mode = AptMode.STUBS_AND_APT
+
+            it.flags.apply {
+                addAll(kaptFlags)
 
-			if (verbose)
-				it.flags.addAll(KaptFlag.MAP_DIAGNOSTIC_LOCATIONS, KaptFlag.VERBOSE)
+                if (verbose) {
+                    addAll(KaptFlag.MAP_DIAGNOSTIC_LOCATIONS, KaptFlag.VERBOSE)
+                }
+            }
 		}
```

Comment:
Oh gosh, tabs vs spaces.

Reply:
Yeah the original project has inconsistencies and I haven't enabled a formatted yet